data/src/main/java/com/microsoft/azure/kusto/data/ClientRequestProperties.java

```diff
@@ -151,8 +152,15 @@ public void clearParameters() {
         parameters.clear();
     }
 
+    /**
+     * Gets the amount of time a query may execute on the service before it times out. Value must be between 1 minute and 1 hour,
+     * and so if the value had been set below the minimum or above the maximum, the value returned will be adjusted accordingly.
+     */
     public Long getTimeoutInMilliSec() throws ParseException {
-        Object timeoutObj = getOption(OPTION_SERVER_TIMEOUT);
+        return getTimeoutInMilliSec(getOption(OPTION_SERVER_TIMEOUT));
+    }
+
+    private static Long getTimeoutInMilliSec(Object timeoutObj) throws ParseException {
```

Comment:
Defining this method allows for DRY, reused by the existing `String getTimeoutAsString(Object timeoutObj)`